codex-rs/core/src/codex.rs

```diff
@@ -1123,7 +1123,28 @@ async fn try_run_turn(
     let mut stream = sess.client.clone().stream(&prompt).await?;
 
     let mut output = Vec::new();
-    while let Some(Ok(event)) = stream.next().await {
```

Comment:
Hmm, so before, if it were `None`, we would just return whatever `output` we had thus far?